.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -11,25 +11,24 @@ body:
       - Search [open](https://github.com/godotengine/godot/issues) and [closed](https://github.com/godotengine/godot/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported. If you don't find a relevant match or if you're unsure, don't hesitate to **open a new issue**. The bugsquad will handle it from there if it's a duplicate.
       - Verify that you are using a [supported Godot version](https://docs.godotengine.org/en/stable/about/release_policy.html).
 
-- type: input
-  attributes:
-    label: Godot version
-    description: >
-      Specify the Git commit hash if using a development or non-official build.
-      If you use a custom build, please test if your issue is reproducible in official builds too.
-    placeholder: 3.3.stable, 4.0.dev (3041becc6)
-  validations:
-    required: true
-
-- type: input
+- type: textarea
   attributes:
-    label: System information
+    label: Environment information
     description: |
       - Specify the OS version, and when relevant hardware information.
+      - Specify the Git commit hash if using a development or non-official build. If you use a custom build, please test if your issue is reproducible in official builds too.
       - For issues that are likely OS-specific and/or graphics-related, please specify the CPU model and architecture.
       - For graphics-related issues, specify the GPU model, driver version, and the rendering backend (GLES2, GLES3, Vulkan).
       - **Bug reports not including the required information may be closed at the maintainers' discretion.** If in doubt, always include all the requested information; it's better to include too much information than not enough information.
-    placeholder: Windows 10, Intel Core i5-7200U, GLES3, Intel HD Graphics 620 (27.20.100.9616)
+      - **Starting Godot 4.1, you can copy this information to your clipboard by going in the Editor to Help > Copy System Info**
+    placeholder: |
+      - OS: Windows 10
+      - Godot Version: 3.3.stable, 4.0.dev (3041becc6)
+      - Rendering Driver: GLES3, Vulkan
+      - Rendering Method: /, forward_plus
```

Comment:
Out of curiosity, why is there a `/` in the output for Rendering Method?

Reply:
It stands for `no info`  
Godot 3 has only one option: forward rendering. And afaik it was (officially) only mentioned in this blog entry: https://godotengine.org/article/godot-3-renderer-design-explained/  
So many Godot 3 users probably don't know/remember the rendering method

Should I change/remove it?

Reply:
I would only list `forward_plus`/`mobile`/`gl_compatibility`, but display them as prettified strings ("Forward+", "Mobile", "Compatibility").

Reply:
Roger that

Additionally, should I adjust the code to attempt to prettify the Rendering Method before copying to clipboard? (would be a simple inline `HashMap<String, String>`)

Reply:
> Additionally, should I adjust the code to attempt to prettify the Rendering Method before copying to clipboard? (would be a simple inline HashMap<String, String>)

We could probably use a `if`/`else if` statement like we do elsewhere already :slightly_smiling_face: 

It might be worth adding a RenderingServer method to render a human-readable rendering method name, but that's better left to a future PR.

Reply:
Alrighty, prettify is done (in bug yml file and code)